client/my-sites/posts/post-total-views.jsx

```diff
@@ -87,18 +46,26 @@ var PostTotalViews = React.createClass( {
 		const site = sites.getSite( siteId );
 
 		return (
-			<a href={ '/stats/post/' + postId + '/' + site.slug }
+			<a href={ `/stats/post/${postId}/${site.slug}` }
 				className={ classNames( {
 					'post__total-views': true,
 					'is-empty': ! viewsCountDisplay
 				} ) }
 				title={ viewsTitle }
 				onClick={ this.props.clickHandler }>
+				<QueryPostStats siteId= { siteId } postId={ postId } stat="views" heartbeat={ HEARTBEAT_IN_SECONDS * 1000 } />
```

Comment:
Lets do rip out the heartbeat here while we are updating this - but leave the support in `QueryPostStats`